program/rust/src/c_oracle_header.rs

```diff
@@ -54,6 +56,22 @@ impl PythAccount for PriceAccount {
     const INITIAL_SIZE: u32 = PC_PRICE_T_COMP_OFFSET as u32;
 }
 
+impl PythAccount for PermissionAccount {
+    const ACCOUNT_TYPE: u32 = PC_ACCTYPE_PERMISSIONS;
+    const INITIAL_SIZE: u32 = size_of::<PermissionAccount>() as u32;
+}
+
+#[repr(C)]
+#[derive(Copy, Clone, Pod, Zeroable)]
+pub struct PermissionAccount {
+    pub header:                  AccountHeader,
+    pub bump:                    u8,
```

Comment:
```suggestion
    pub bump_seed:               u8,
```

Reply:
What’s the reason we’re storing the bump seed?

Reply:
Small efficiency gain when verifying PDAs. Although I ended up deciding against it.